lib/ret/media_resolver.ex

```diff
@@ -391,4 +379,28 @@ defmodule Ret.MediaResolver do
   defp module_config(key) do
     Application.get_env(:ret, __MODULE__)[key]
   end
+
+  defp ytdl_resolution(%MediaResolverQuery{low_resolution: true}), do: "480"
+  defp ytdl_resolution(_query), do: "720"
+
+  defp ytdl_query(query, "crunchyroll.com") do
+    resolution = query |> ytdl_resolution
+    ext = query |> ytdl_ext
+
+    # Prefer a version with baked in (english) subtitles. Client locale should eventually determine this
+    "best#{ext}[format_id*=hardsub-enUS][height<=?#{resolution}]/best#{ext}[format_id*=hardsub-enUS]/" <>
+      ytdl_query(query, nil)
+  end
+
+  defp ytdl_query(query, _root_host) do
+    resolution = query |> ytdl_resolution
+    ext = query |> ytdl_ext
+
+    "best#{ext}[protocol*=http][height<=?#{resolution}]/best#{ext}[protocol*=m3u8][height<=?#{resolution}]/best#{ext}[protocol*=http]/best#{
```

Comment:
These are starting to get a tad unwieldily, might be nice to make a list and join with "/", but not a huge deal